submissions/ses3332002/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,415 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Homework01</title>
+  </head>
+
+  <body>
+    <header>
+      <h1>Наши котики</h1>
+      <img class="poster" src="./img/poster.jpg" alt="постер" />
+      <p class="session_date">04.02.2020</p>
```

Comment:
Great film.
For date and time preferable to use `<time>` tag [Read more](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time)

Reply:
I've remade it